agent/config/ank-agent.conf

```diff
@@ -0,0 +1,46 @@
+# ank-agent.conf is the configuration file for an Ankaios agent.
+# This configuration file is formatted in the TOML language.
+
+# The version of the configuration file.
+version = 'v1'
+
+# The name to use for the registration with the server.
+# Every agent has to register with a unique name.
+# Agent name shall contain only regular upper and lowercase characters
+# (a-z and A-Z), numbers and the symbols "-" and "_".
+# name = 'agent_1'
+
+# The server url.
+# If started in insecure mode then the HTTP protocol shall be used,
+# otherwise the HTTPS protocol shall be used.
+# server-url = 'https://127.0.0.1:25551'
+
+# An existing path where to manage the fifo files.
+# run-folder = '/tmp/ankaios/'
+
+# The flag to disable TLS communication between
+# the Ankaios server, agents and the ank CLI.
+# If set to 'true', then the agent shall not use TLS.
+# insecure = 'false'
+
+# The path to agent ca certificate pem file.
```

Comment:
```suggestion
# The path to the ca certificate pem file.
```

There is no specific CA file for the agent. There is just one CA file which is shared between server, agent and CLI.